web/src/reducers/dataReducer.js

```diff
@@ -182,8 +181,6 @@ module.exports = (state = initialDataState, action) => {
           [action.zoneId]: action.payload.map(datapoint => ({
             ...datapoint,
             hasParser: true,
-            // Exchange information is not shown in history observations without production data, as the percentages are incorrect
-            exchange: isEmpty(datapoint.production) ? {} : datapoint.exchange,
```

Comment:
What's the reason for removing `exchange` here altogether? Where did it show? :)

Reply:
Ah nevermind, I now realise we already do `...datapoint` above!

Reply:
`exchange` is still there, but now we do not modify it in the reducer. Less remapping.